src/plugins/embeddable/public/components/panel_options_menu/__examples__/panel_options_menu.stories.tsx

```diff
@@ -17,37 +17,45 @@
  * under the License.
  */
 
-import * as React from 'react';
-import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
-import { withKnobs, boolean } from '@storybook/addon-knobs';
+import * as React from 'react';
```

Comment:
we can use the default import here`import React from 'react';`. It shouldn't change anything else in this file.

Reply:
VSCode auto-formatted this for me so I thought maybe this plugin had some lint rules around this. @elastic/kibana-app-arch let me know if there's a preference for react imports here.